include/dda.h

```diff
@@ -0,0 +1,38 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   dda.h                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ycarro <ycarro@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/22 14:48:30 by ycarro            #+#    #+#             */
+/*   Updated: 2023/02/23 14:15:17 by ycarro           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef DDA_H
+# define DDA_H
+
+# include <stdlib.h>
+# include <stdio.h>
+# include <math.h>
+
+# include "libft.h"
+# include "angle.h"
+# include "point.h"
+
+typedef struct s_ray
+{
+	t_point	x_pos;
```

Comment:
i think x_pox and y_pox are not clear names
are you referring to the next y/x point?
if se then maybe call them next_x and next_y

Reply:
They are called like that becouse they are like the actual position of the steps on x and y, but yes maybe I can find a better name